tests/admin.reported.comment.test.js

```diff
@@ -0,0 +1,65 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../src/index';
+import status from '../src/helpers/constants/status.codes';
+
+
+let userToken = '';
+
+const { expect } = chai;
+const password = 'password23423';
+
+chai.use(chaiHttp);
+
+describe('test the controller for an admin to get reported comment', () => {
+  it('should login the moderetor to get the token', (done) => {
```

Comment:
@MCFrank16 can we create a helper method for logging in regular users, which everyone can use in their tests?
@alainburindi created one such helper method for admins a while back. It would be cool if that method can be updated to sign in both admins and regular users.

Reply:
cool, lemme put my hands on it.